cms/templates/admin/cms/page/change_form.html

```diff
@@ -42,7 +42,7 @@
 
 {% block object-tools %}{% endblock %}
 
-<form {% if has_file_field %}enctype="multipart/form-data" {% endif %}action="?language={{ language }}{%if request.GET.parent_node %}&amp;parent_node={{ request.GET.parent_node }}{% endif %}{%if request.GET.source %}&amp;source={{ request.GET.source }}{% endif %}" method="post" id="page_form">
```

Comment:
Wondering if these changes for &amp; are actually a separate bug / PR?

Reply:
Right. This is not connected to the rest. It just came to mind that Django since 2.2.19 does not support semicolons as separators in urls any more (https://docs.djangoproject.com/en/4.1/releases/2.2.19/) Hence it does not understand `&amp;` as a separator. However, this seems not critical for the test. It passes w/o change.